Correct wording of deprecation in Registration (Satellite) (#2123)
theforeman/foreman-documentation: guides/common/modules/con_registering_hosts.adoc

NOTE: Red{nbsp}Hat Enterprise{nbsp}Linux versions 6.1, 6.2, and 6.3 require `subscription-manager` and related packages to be updated manually.
For more information, see https://access.redhat.com/solutions/4480641[].

Note that the entitlement-based subscription model is deprecated and will be removed in a future release.

> {Team} recommends that you use the access-based subscription model of https://access.redhat.com/articles/simple-content-access[Simple Content Access] as a substitute.

Reviewer: Maybe this simplifies a little? Consider optional.

```suggestion
{Team} recommends that you use the access-based subscription model of https://access.redhat.com/articles/simple-content-access[Simple Content Access] instead.
```
